### ISSUE TITLE
Incorrect error message for negative of numpy.sum

### ISSUE THREAD

**Issue by igordertigor** [python/mypy#13958]
**Bug Report**

Not sure if this is rather an issue with numpy. I'm therefore posting this in both numpy and here:

When I take the negative of the sum of a array's elements by calling the `numpy.sum` function with a minus sign before (i.e. `-numpy.sum(x)`), I get the following error from mypy:
```
test.py:5: error: Unsupported operand type for unary - (<nothing>)
Found 1 error in 1 file (checked 1 source file)
```
However, calling the `.sum`-method on the array (which should be equivalent) gives no error. In both cases, the code works just fine. See below for code example

**To Reproduce**

```python
import numpy as np


def fail(p: np.ndarray) -> float:
    return -np.sum(p)


def ok(p: np.ndarray) -> float:
    return -p.sum()


if __name__ == '__main__':
    print(fail(np.ones(3)))
    print(ok(np.ones(3)))
```

**Expected Behavior**

I would expect to errors for the above script and a message:
```
Success: no issues found in 1 source file
```

**Actual Behavior**

```
test.py:5: error: Unsupported operand type for unary - (<nothing>)
Found 1 error in 1 file (checked 1 source file)
```

**Your Environment**

- Mypy version used: mypy 0.982 (compiled: yes)
- Mypy command-line flags: none (except the source file)
- Mypy configuration options from `mypy.ini` (and other config files):
```
[mypy]
plugins = numpy.typing.mypy_plugin
strict_optional = True
ignore_missing_imports = True
follow_imports = skip
```
- Python version used: 3.8.10

**Comment by igordertigor** [comment]
I didn't install the numpy stubs, just numpy (version 1.23.4).

**Comment by igordertigor** [comment]
Also, here is the corresponding numpy issue: https://github.com/numpy/numpy/issues/22495 They say it's more likely an issue with mypy than with numpy. Specifically, it seems to be an issue with my `mypy.ini`:
```
[mypy]
plugins = numpy.typing.mypy_plugin
strict_optional = True
ignore_missing_imports = True
follow_imports = skip
```
It seems that mypy is incorrectly resolving the follow_imports setting here.
